frontend/src/metabase-lib/parameters/utils/click-behavior.unit.spec.ts

```diff
@@ -83,9 +84,9 @@ describe("metabase/lib/click-behavior", () => {
 
     it("should include user attribute values from extraData", () => {
       expect(
-        getDataFromClicked({ extraData: { userAttributes: { foo: "bar" } } })
+        getDataFromClicked({ extraData: { userAttributes: ["foo", "bar"] } })
           .userAttribute,
-      ).toEqual({ foo: { value: "bar" } });
+      ).toEqual({ foo: { value: "foo" }, bar: { value: "bar" } });
```

Comment:
This test has been updated because previously code assumed that `userAttributes` is an object, not an array.
I'm not 100% sure whether the expected value is actually expected.